Assets/HighScoreScript.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class HighScoreScript : MonoBehaviour
+{
+    public GameObject score;
+    public GameObject name;
```

Comment:
I get following warning, perhaps we should adjust to it.
```
Assets\HighScoreScript.cs(9,23): warning CS0108: 'HighScoreScript.name' hides inherited member 'Object.name'. Use the new keyword if hiding was intended.
```

Reply:
Added 'new' before GameObject